### PR TITLE
feat: biw polish

### LINKED ISSUE #1551: BIW: Editor movement with the middle button of the mouse is not working correctly and biw polishing
If you drag the mouse while holding the middle button (roulette) the camera is not working as expected, since it moves too fast

### PR DESCRIPTION
**Pull request by Suduck**
## What does this PR change?

This PR polish the biw tool and fix #1551 

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/biw-polish&kernel-branch=feat/biw-invalidate-scene-info
2. Test builder in world completly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Comment by lorux0** [comment]
there is an odd delay for the highlight feedback whenever you hover an item with the mouse

**Comment by lorux0** [comment]
missing graphic at controls menu:
Screenshot 2022-03-30 095553

**Comment by lorux0** [comment]
when you hide the ui, i forgot the shortcut and i found no feedback about how to re-enable it.. i had to refresh

**Comment by lorux0** [comment]
There is an issue about activating/deactivating the portable experience when getting in/out of biw:
1. wear a portable experience
2. enter biw
3. exit biw
4. the portable experience is deactivated (when it should be active)
5. enter biw
6. the portable experience is activated inside biw

Screenshot 2022-03-30 104548

**Comment by lorux0** [comment]
missing graphic for `x` on the project search bar
Screenshot 2022-03-30 110155

**Comment by Suduck** [comment]
> there is an odd delay for the highlight feedback whenever you hover an item with the mouse

This is done for optimization purposes

> for some reason i dont get tris, materials or texture information about the tree asset.. its the only one in my scene with this problem Screenshot 2022-03-30 095708

This is a know bug, this needs to be fixed by the dApps team

**Comment by Suduck** [comment]
> there is an ui glitch when you duplicate or delete a project. Also when it gets duplicated it not copying the thumbnail
> 
>  2022-03-30.10-57-39.mp4

This is not a glich, you see the loading panel view for a momment ( because they are loading the new projects) and then the projects, I think it is a weird loading screen too

File: unity-renderer/Assets/DCLPlugins/BuilderInWorld/Scripts/SceneManager/SceneManager.cs [hunk 1]
```diff
@@ -561,6 +563,9 @@ internal void EnterEditMode()
 
             DCLCharacterController.OnPositionSet += ExitAfterCharacterTeleport;
 
+            portableExperiencesToResume = DataStore.i.experiencesViewer.activeExperience.Get();
```
[the remark this answers is not in the document]
Comment (Suduck) [reply unity-renderer/Assets/DCLPlugins/BuilderInWorld/Scripts/SceneManager/SceneManager.cs:566]: That would require that the DataStore has an interface and it still doesn't have it. I remember that you are implementing one but couldn't find it

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExperiencesViewer/Scripts/ExperiencesViewerComponentController.cs [hunk 2]
```diff
@@ -163,11 +163,12 @@ public void OnPEXSceneAdded(IParcelScene scene)
             }
 
             GlobalScene newPortableExperienceScene = scene as GlobalScene;
-
+            DataStore.i.experiencesViewer.activeExperience.Get().Add(scene.sceneData.id);
```
[the remark this answers is not in the document]
Comment (Suduck) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExperiencesViewer/Scripts/ExperiencesViewerComponentController.cs:166]: That would require that the DataStore has an interface and it still doesn't have it. I remember that you are implementing one but couldn't find it

File: unity-renderer/Assets/DCLPlugins/BuilderInWorld/Scripts/SceneManager/SceneManager.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (lorux0) [reply unity-renderer/Assets/DCLPlugins/BuilderInWorld/Scripts/SceneManager/SceneManager.cs:566]: there is no need for an interface, you can just inject `DataStore` object

**Pull request merged**